plugins/catalog-backend/src/ingestion/types.ts

```diff
@@ -54,6 +54,17 @@ export type LocationReader = {
   read(location: LocationSpec): Promise<ReadLocationResult>;
 };
 
+// TODO: Is this a candidate for the catalog model? I don't think so because it contains both metadata (but not all metadata) and the content itself!
```

Comment:
Yeah should the data be in the model at all? :) I was thinking the data itself was always out of band, from a consumption standpoint